plugins/plugin-site/src/utils/algolia-queries.js

```diff
@@ -12,6 +12,16 @@ function pluginQueries() {
               currentInstalls
               trend
             }
+            healthScore {
+                value
+                    version
+                    details {
+                        key
+                        value
+                        coefficient
+                    }
+                    timestamp
```

Comment:
![image](https://user-images.githubusercontent.com/110087/230733839-e365e25f-c654-4b98-856e-47545215fdee.png)

can you fix the alignment here?